zerver/openapi/zulip.yaml

```diff
@@ -1459,7 +1459,28 @@ paths:
                                 value:
                                   description: |
                                     The new value of the changed property.
+
+                                    **Changes**: Starting with Zulip 10.0 (feature level 320), this
+                                    field can be an object for `can_remove_subscribers_group` property
+                                    when the setting is set to a combination of users and groups.
```

Comment:
This should probably mention and link group-setting value.

Reply:
Since the value already supported integer type and group-setting value is union of integer and object, am not sure we should just mention that `can_remove_subscribers_group` is now a group-setting value. So, just updated the text accordingly.